src/common/tiles/BigTile/styled.js

```diff
@@ -232,14 +250,15 @@ export const Article = styled.article`
   line-height: 160%;
   color: ${({ theme }) => theme.colors.black};
   margin: 0;
+  
   @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
     font-size: 17px;
     grid-row: 3;
     grid-column: span 2;
-    margin-top: 36px;
+    margin-top: 24px;
   }
   @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
```

Comment:
Tu dał bym drobny padding, bo tekst na mobi jest zbyt przyklejony do górnej treści

Reply:
To jest temat o ktorym rozmawialismy na callu, moim zdaniem tez powinien byc jakis maly padding po to, zeby na wiekszych niz 320px byla jednak jakas odleglosc tekstu od zdjecia, ale zmienilem to za wasza sugestia, ale nie ma problemu dodam temu jakis padding :P